src/controls/combinators/OneOfControl.tsx

```diff
@@ -34,26 +34,28 @@ export function OneOfControl({
     uischemas,
   )
 
-  const combinatorSchemaSwitcher = (
-    <CombinatorSchemaSwitcher
-      config={config as unknown}
-      renderInfos={combinatorRenderInfos}
-      selectedIndex={selectedIndex}
-      setSelectedIndex={setSelectedIndex}
-      uischema={uischema}
-      path={path}
-      // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
-      data={data}
-      handleChange={handleChange}
-      rootSchema={rootSchema}
-    />
-  )
-
   return (
     <Space direction="vertical" style={{ width: "100%" }} size="middle">
-      <ControlLabel uischema={uischema as ControlUISchema} schema={schema} />
-      {combinatorSchemaSwitcher}
-      {combinatorRenderInfos.map((renderInfo, index) => {
+      {uischema.type === "Control" && ( // I don't think it's possible for this to be false
+        // but until we improve the UISchema types a bit, it's hard to be sure
+        <ControlLabel
+          uischema={uischema as ControlUISchema<unknown>}
+          schema={schema}
+        />
+      )}
+      <CombinatorSchemaSwitcher
+        config={config as unknown}
+        renderInfos={oneOfRenderInfos}
+        selectedIndex={selectedIndex}
+        setSelectedIndex={setSelectedIndex}
+        uischema={uischema}
+        path={path}
+        // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
+        data={data}
+        handleChange={handleChange}
+        rootSchema={rootSchema}
+      />
+      {oneOfRenderInfos.map((renderInfo, index) => {
```

Comment:
Does this change belong in this PR? I ended up doing something else in [this open PR](https://github.com/great-expectations/jsonforms-antd-renderers/pull/55/files#diff-0a6875817fdfb9356e8575e3b1b5130af4ce7a08bb05802911fc47660bba9722R56).

Reply:
This is really a reversion of changes from #56, since it seemed to include [changes to OneOf that weren't necessary for that PR](https://github.com/great-expectations/jsonforms-antd-renderers/pull/56/files#diff-0a6875817fdfb9356e8575e3b1b5130af4ce7a08bb05802911fc47660bba9722R37), so when I was resolving merge conflicts, keeping the previous code was a lot simpler.